### PR TITLE
feat: [FFM-2259]: Add governence tab to cf module

### PR DESCRIPTION
**Pull request by conormurray95**
##### Summary:
Added governance tab to cf module

##### Screenshots:
Screenshot 2022-02-17 at 14 34 32

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by conormurray95** [comment]
trigger cypress
retrigger titlecheck

**Comment by conormurray95** [comment]
trigger cypress

**Comment by conormurray95** [comment]
trigger cypress

**Review by kylekstewart — APPROVED** [review]

File: src/modules/75-cf/RouteDestinations.tsx [hunk 1]
```diff
@@ -431,6 +432,11 @@ const CFRoutes: FC = () => {
       </RouteWithLayout>
       <AdminRouteDestinations />
       <PipelineRouteDestinations />
+
+      {GovernanceRouteDestinations({
+        sidebarProps: CFSideNavProps,
+        pathProps: { ...accountPathProps, ...projectPathProps, ...cfModuleParams }
+      })}
```
Comment (knagurski) [inline src/modules/75-cf/RouteDestinations.tsx:439]: I'm pretty sure we don't need to do this or change this file at all. The routing for the governance app should already be handled in the main app RouteDefinitions (`@common/RouteDefinitions`)

Comment (conormurray95) [reply src/modules/75-cf/RouteDestinations.tsx:439]: The common RouteDefinitions just enables the global account settings governance tab, all the modules that use it also define this bit in their own RouteDefinitions files for the module specific tab

**Review by crystal-fong — APPROVED** [review]
LGTM!

**Comment by knagurski** [comment]
LGTM

**Pull request merged**